### PR TITLE
rebased PR#6

### PR DESCRIPTION
**Pull request by livnev**
This is PR #6 rebased on top of latest `lev`. **12** behaviours are now passing. :smile_cat:

Please review and merge.

### CODE REVIEW

**Comment by mhhf** [comment]
plz remove dach stuff from lemmas and storage, then i'm fine with merging :+1:

**Comment by MrChico** [comment]
Can you add the passing specs to the testpassing in the makefile so we can test them easily and make sure we don't break anything

**Comment by livnev** [comment]
@MrChico @mhhf I've added the passing behaviours to the `Makefile`, let's see if they can complete overnight! :smiley: 

EDIT: uh oh:

```
cd dss/ && dapp clean && cd ../
/bin/sh: 1: dapp: not found
Makefile:34: recipe for target 'dapp-clean' failed
make: *** [dapp-clean] Error 127
program finished with exit code 2
elapsedTime=0.007106
```

File: abi2specs [hunk 1]
```diff
@@ -221,7 +226,7 @@ let behaviours = acts
       .map(name => {
         let code = "";
         if(testPath("dss/out/" + name + ".bin-runtime")) {
-          let code = fs.readFileSync("dss/out/" + name + ".bin-runtime") // TODO
+            code = fs.readFileSync("dss/out/" + name + ".bin-runtime").toString()
```
Comment (mhhf) [inline abi2specs]: this needs to be done dynamically based on the given json, not on .bin-runtime

**Pull request closed without merging**